media/css/core.css

```diff
@@ -690,19 +690,16 @@ div.project-import-remote button.remote-sync:before {
     content: "\f021";
 }
 
-div.project-import-remote form.import-connect-github button:before,
 a.socialaccount-provider.github:before {
     font-family: FontAwesome;
     content: "\f09b";
 }
 
-div.project-import-remote form.import-connect-gitlab button:before,
 a.socialaccount-provider.gitlab:before {
     font-family: FontAwesome;
     content: "\f1d3";
```

Comment:
`\f296` should be for gitlab
http://fontawesome.io/icon/gitlab/

Reply:
Nice!